src/activities/solar_system/QuizScreen.qml

```diff
@@ -0,0 +1,221 @@
+                height: background.horizontalLayout ? background.height - bar.height - closenessMeter.height - questionArea.height - 50 * ApplicationInfo.ratio
+                        : parent.itemHeightVertical
+                spacing: 10 * ApplicationInfo.ratio
+                orientation: Qt.Vertical
+                verticalLayoutDirection: ListView.TopToBottom
+                interactive: false
+                model: optionListModel
+
+                property int buttonHeight: height / 4 * 0.9
+
+                add: Transition {
+                    NumberAnimation { properties: "y"; from: parent.y; duration: 500 }
+                }
+
+                delegate: Item {
+                    id: optionListViewDelegate
+                    width: optionListView.width
+                    height: optionListView.buttonHeight
+                    anchors.right: parent.right
+                    anchors.left: parent.left
+
+                    property int closenessValue: closeness
+
+                    AnswerButton {
+                        id: optionRectangle
+                        width: parent.width
+                        height: optionListView.buttonHeight
+                        textLabel: optionValue
+                        anchors.right: parent.right
+
+                        isCorrectAnswer: closenessValue === 100
+                        onIncorrectlyPressed: {
+                            if(!background.assessmentMode) {
+                                if(correctAnswerAnim.running)
+                                    correctAnswerAnim.stop()
+                                if(incorrectAnswerAnim.running)
+                                    incorrectAnswerAnim.stop()
+                                incorrectAnswerAnim.start()
+                                mainQuizScreen.closenessValueInMeter = closenessValue
+                            }
+                        }
+                        onCorrectlyPressed: {
+                            if(!background.assessmentMode) {
+                                if(correctAnswerAnim.running)
+                                    correctAnswerAnim.stop()
+                                if(incorrectAnswerAnim.running)
+                                    incorrectAnswerAnim.stop()
+                                particles.burst(30)
+                                correctAnswerAnim.start()
+                                mainQuizScreen.closenessValueInMeter = closenessValue
+                            }
+                            else
+                                Activity.nextSubLevel()
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    Score {
+        id: score
+        anchors.bottom: undefined
+        anchors.right: parent.right
+        anchors.rightMargin: 10 * ApplicationInfo.ratio
+        anchors.top: parent.top
+    }
+
+    Rectangle {
```

Comment:
this is behind the bar, is there a way to move it so it won't be overriden by it?

Reply:
Fixed. :+1: